shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -78,7 +78,9 @@
           .hasArg(true)
           .numberOfArgs(1)
           .argName("hosts")
-          .desc("A list of worker hosts separated by comma")
+          .desc("A list of worker hosts separated by comma, when host and locality lists are empty,"
+              + " all hosts will be selected unless excluded by setting excluded-hosts or"
+              + " excluded-locality")
```

Comment:
What's the relationship between the hosts options and locality options? are they actually affect each other?

Reply:
No, they don't affect each other. When they are not set, all worker will be chosen.